src/box/lua/config/instance_config.lua

```diff
@@ -715,6 +715,12 @@ return schema.new('instance_config', schema.record({
             box_cfg = 'memtx_max_tuple_size',
             default = 1024 * 1024,
         }),
+        sort_threads = schema.scalar({
+            type = 'number',
```

Comment:
I doubt that a fractional part has any meaning here. So, I would mark it as `'integer'`.

Reply:
Done.